solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseHttpClusterStateProvider.java

```diff
@@ -124,7 +119,10 @@ private ClusterState fetchClusterState(
       throws SolrServerException, IOException, NotACollectionException {
     ModifiableSolrParams params = new ModifiableSolrParams();
     if (collection != null) {
+      log.debug("Making a call to Solr to fetch cluster state for collection: ", collection);
```

Comment:
Needs the if debug check...

Reply:
Done

Reply:
Why do you say that Eric?  If it was needed, the precommit validation would tell her.  Part of the value of the PR validation is that we don't need to waste any time telling contributors something that the automation will inform them to do.  (similar for code formatting with tidy)  In this case, the if-check is *not* needed because this log call is simple -- no method call inside the arguments.  If the parameter there was foobar.callingAMethod().toString() or some-such, *then* the precommit check would say to add a condition.  Even though the verbosity of adding two lines for the isDebugEnabled check is minor, it annoys me -- I don't want to see it added if the validation doesn't ask us to.

Reply:
I am relying on logs in HttpSolrCall and removed the logging in BaseHttpCSP.